vars/getBranchesFromAliases.groovy

```diff
@@ -60,3 +65,28 @@ def getBranchNameFromAlias(alias) {
   }
   return alias
 }
+
+def subtractBranchIfPossible(String branch, String subtrahend) {
```

Comment:
If there is no collision with other method (i.e. one in which it's always possible), I'd prefer a simpler name
```suggestion
def subtractBranch(String branch, String subtrahend) {
```

Reply:
No strong opinion. My aim was to be sure the name could clarify the expectation, it's not a normal subtraction, since the outcome could contain a negative minor, and if so, it will return the branch by default.

Let me rename it

Reply:
Not a blocker on my side though :D